hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -50,6 +50,43 @@
       milestone: V2
       tests: ["pattgen_perf"]
     }   
+    {
+      name: rand
+      desc: '''
+            Checking ip operation with random counter values
+
+            Stimulus:
+              - Program the pre-divider and size registers to unconstraint random values
+              - Program the clk_cnt, bit_cnt and rep_cnt to values less than but close to 
+		predivider and size registers, so that counting would take a reasonable 
+		number of clock cycles
+	      - include programming for corner cases
+                repeat programming a random number of times
+              - Start and stop channels quickly
+              - Clear interrupts quickly
+            Checking:
+              - Ensure patterns are correctly generated
+              - Ensure interrupts are robust asserted and cleared (e.g. at the high data rate)
+            '''
+      milestone: V2
+      tests: ["pattgen_rand"]
+    }   
+    {
+      name: alert
+      desc: '''
+            Checking alert errors
+
```

Comment:
Write todo comment to identify exactly alerts are triggered, and fot the V2 find out what exactly what triggers alert, if anything